### ISSUE TITLE
UI - there is something wrong with the results list

### ISSUE THREAD

**Issue by cryodream** [theotherp/nzbhydra2#355]
This started happening yesterday.
I am using **[linuxserver/hydra2 - Docker Hub](https://hub.docker.com/r/linuxserver/hydra2/)** container in unRAID.

1. The results list is messed up - it shows only a small fraction of results.
    * Notice, that UI says: **Loaded 200 of 278 results (rejected 0)**. Then, why does it shows only **12** results? The first page has 7, the second page has 5.
    * Also, you can clearly see in the console - there are 200 results loaded.

2. Why is the list pagerized into 2 pages, with **only** 12 results?
    * And some weird number per page - 7? Where did that come from?

3. In the second screenshot - I have selected to filter on **1080p**.
    * Now the result list shows 10 results in the same page! Not 7 😕 
    * I **know** there are more than 10 results, that this filter should show, if it actually worked properly, and filtered the whole 200 results. So what is going on? What is it filtering?

### Basically, nzbhydra has just become unusable, because I cannot trust it's search/results whatsoever.

How do fix it?

![2019-04-10_14-09-55__chrome](https://user-images.githubusercontent.com/844036/55903717-4d373400-5b9c-11e9-8608-52a2c0e5c065.png)

![2019-04-10_14-23-39__chrome](https://user-images.githubusercontent.com/844036/55903722-50322480-5b9c-11e9-9fd3-4b6a75cde288.png)

**Comment by theotherp** [comment]
The results are sorted by season and episode. Click the plus symbol to expand the group or in the display options disable "Group TV results by season/episode".

But I agree there's still something wrong with the display.

I recommend reading the changelog where I inform about new features...

**Comment by cryodream** [comment]
@theotherp thank you.
This is quite a major change to enable by default, without a HUGE warning 😄 
I mean, who reads all the release notes, every single time? Especially, using docker 😉 

I have disabled the grouping, because as it stands, it makes it even harder and less convenient to filter and select particular nzbs from the results list.

Oh, and probably you should not do paging on the grouped results, as that makes that much more confusing and inconvenient. Just a thought.

**Comment by theotherp** [comment]
I fixed the grouping. 

If something has worked until now and with a new version it suddenly doesn't it means something has changed. A look into the changelog would've explained that.

Anyway, disable the feature if you want and you're good.
